### PR TITLE
Update Fidelity documentation

### PR DESCRIPTION
**Pull request by mathstuf**
Not sure if there still exist account types which are not supported or
not. SMS-only though when through this account type though.

### CODE REVIEW

File: notes/fidelity.html [hunk 1]
```diff
@@ -12,12 +12,7 @@ <h2 class="ui header">
         For <em>authentication</em>, Fidelity uses either a physical token provided by Fidelity
         or the Symantec VIP Access smartphone app.
```
Comment (VictorYing) [inline notes/fidelity.html]: The first sentence up here needs updating.

File: notes/fidelity.html [hunk 2]
```diff
@@ -71,6 +66,11 @@ <h2 class="ui header">
         <li>
             Call Fidelity Account Key Support at +1 800-544-6666.
         </li>
+        <li>
+            Once logged in, navigate to <b>Profile</b> &gt; <b>Security
+            Center</b> &gt; <b> 2-factor authentication</b>. This process only
+            supports SMS.
```
Comment (VictorYing) [inline notes/fidelity.html]: I have an ordinary personal account at Fidelity.com (not set up through an employer). I signed up for 2FA with my phone number. When I log in via an incognito/private browser I get prompted after my password for whether I'd like a text message or automated voice call to receive a security code, so it seems it isn't true that this only supports SMS.

File: notes/fidelity.html [hunk 3]
```diff
@@ -12,12 +12,7 @@ <h2 class="ui header">
...
-        Fidelity only offers 2FA on certain account types. For example, they
-        do not support it for institutional retirement accounts of the kind
-        that might be set up through an employer.
-        <br>
-        It appears there is no information about this service provided
-        online and it must be enabled and managed by phone.
+        Fidelity may only offer 2FA on certain account types.
     </p>
 
     <p>
```
Comment (VictorYing) [inline notes/fidelity.html]: I believe everything below here about "As of October 2016..." and "That this is different than the weaker Transaction Security..." is outdated. The two links below still link to pages that exist, but the page content has changed and the quoted text is no longer there, and Fidelity is no longer advertising Transaction Security as a feature, instead advertising 2FA at login. I'd recommend replacing all this outdated text with a simple link to Fidelity's newer page documenting 2FA: https://www.fidelity.com/security/extra-security-login

File: notes/fidelity.html [hunk 4]
```diff
@@ -71,6 +66,11 @@ <h2 class="ui header">
         <li>
             Call Fidelity Account Key Support at +1 800-544-6666.
         </li>
+        <li>
```
Comment (VictorYing) [inline notes/fidelity.html]: The organization of these nested lists is rather confusing. As documented at https://www.fidelity.com/security/extra-security-login, Fidelity has two separate routes to two different classes of 2FA for logins. You can either set up the Symantec VIP Access proprietary 2FA on logins as before (using either a physical token or the proprietary apps) which still requires a phone call to Fidelity's support line and cannot be set up entirely online, OR you can set up 2FA via a phone number (SMS or automated voice call) which should be set up through the online security center and does not require a proprietary app or physical token.

It's not obvious to me whether it's better to try to document all these details here, or to simply link to (or quote directly from?) the page at Fidelity.com which seems to cover it pretty well: https://www.fidelity.com/security/extra-security-login

**Review by VictorYing — COMMENTED** [review]
It seems much of the information here is outdated and no longer matches my recent experience with 2FA on my Fidelity.com account.

**Comment by kmpoppe** [comment]
@VictorYing thank you for your comments.
From what I read it would seem that the usefulness of our local documentation has ended and we can just link the document of fidelity, maybe with adding some exception text.
I'll look into this later.
Thank you again for your contribution!
// Kai

File: _data/investing.yml [hunk 5]
```diff
@@ -50,7 +50,8 @@ websites:
     url: https://www.fidelity.com
     img: fidelity.png
     tfa:
-      - software
+      - proprietary
+      - sms
       - hardware
     doc: "/notes/fidelity/"
```
Comment (kmpoppe) [inline _data/investing.yml]:
```suggestion
    doc: https://www.fidelity.com/security/how-two-factor-authentication-works
```

Comment (kmpoppe) [reply _data/investing.yml]: @mathstuf Ben,
please replace this link to the page of fidelity and delete the `notes/fidelity.html` files.
Thanks!
// Kai

File: _data/investing.yml [hunk 6]
```diff
@@ -50,7 +50,8 @@ websites:
     url: https://www.fidelity.com
     img: fidelity.png
     tfa:
-      - software
+      - proprietary
+      - sms
       - hardware
```
Comment (kmpoppe) [inline _data/investing.yml:56]:
```suggestion
      - phone
      - hardware
```

Comment (kmpoppe) [reply _data/investing.yml:56]: As 2FA pins can be obtained by text or call, this should be included as well.

**Review by kmpoppe — CHANGES_REQUESTED** [review]
See inline comments

**Comment by mathstuf** [comment]
Thanks @VictorYing and @kmpoppe. I've updated it and left an exception noting that support may vary between different account types (at least I didn't see anything supporting hardware for my account type).

**Review by kmpoppe — APPROVED** [review]

**Comment by kmpoppe** [comment]
Closes #4073

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**